contracts/kleros/TalentLayerMultipleArbitrableTransaction.sol

```diff
@@ -1,106 +1,31 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.9;
 
-import "./Arbitrator.sol";
-import "./IArbitrable.sol";
-import "../interfaces/IJobRegistry.sol";
-
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
-import {ITalentLayerID} from "../interfaces/ITalentLayerID.sol";
+import "hardhat/console.sol";
 
-contract TalentLayerMultipleArbitrableTransaction is IArbitrable {
-    // **************************** //
-    // *    Contract variables    * //
-    // **************************** //
+import "../interfaces/IJobRegistry.sol";
+import "../interfaces/ITalentLayerID.sol";
+import "./IArbitrable.sol";
+import "./Arbitrator.sol";
 
-    uint8 constant AMOUNT_OF_CHOICES = 2;
-    uint8 constant SENDER_WINS = 1;
-    uint8 constant RECEIVER_WINS = 2;
+contract TalentLayerMultipleArbitrableTransaction {
 
-    enum Party {
-        Sender,
-        Receiver
-    }
-    enum Status {
-        NoDispute,
-        WaitingSender,
-        WaitingReceiver,
-        DisputeCreated,
-        Resolved
-    }
+    // =========================== Enum ==============================
 
-    struct WalletFee {
-        address payable wallet;
-        uint fee;
-    }
+    // =========================== Struct ==============================
 
     struct Transaction {
```

Comment:
It might have an impact on the integration with dispute resolution. We can either a) have the attributes in the struct that are used in the standard but then we include attributes that are neither validated or meaningful in our current implementation, or we can add them once we implement dispute resolution.